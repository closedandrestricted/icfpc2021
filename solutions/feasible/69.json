{"vertices":[[32,75],[45,102],[12,125],[14,93],[28,97],[22,126],[18,108],[43,54],[22,124],[9,99],[13,112],[13,116],[20,106],[6,111],[35,108],[17,113],[43,103],[10,120],[23,114],[40,114],[25,108],[10,113],[5,113],[34,79],[6,107],[18,124],[14,111],[5,118],[20,103],[15,87],[7,113],[25,127],[3,119],[35,107]]}