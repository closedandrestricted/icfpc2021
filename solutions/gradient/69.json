{"vertices":[[78,11],[64,39],[106,49],[106,6],[79,44],[113,41],[17,63],[54,10],[34,60],[64,56],[11,66],[8,64],[8,89],[9,73],[94,76],[10,96],[73,9],[14,118],[12,103],[24,81],[54,73],[18,125],[17,111],[29,29],[19,117],[54,92],[56,106],[40,85],[27,105],[106,106],[44,81],[67,39],[71,104],[49,49]]}