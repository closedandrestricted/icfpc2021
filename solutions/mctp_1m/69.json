{"vertices":[[30,64],[61,68],[31,17],[20,87],[51,52],[36,26],[20,65],[19,86],[33,56],[33,58],[21,71],[28,69],[15,66],[25,77],[44,38],[9,63],[47,105],[21,80],[11,70],[16,68],[20,69],[19,87],[23,88],[15,112],[23,94],[36,62],[50,64],[23,64],[31,83],[76,35],[23,58],[21,107],[65,67],[15,86]]}