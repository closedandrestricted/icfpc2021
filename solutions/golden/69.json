{"vertices":[[43,27],[68,42],[110,29],[62,8],[86,36],[118,22],[20,63],[36,4],[37,60],[67,39],[18,69],[10,63],[12,88],[11,72],[104,58],[9,95],[91,11],[13,117],[11,102],[36,84],[50,49],[20,120],[21,114],[30,30],[27,116],[53,66],[61,76],[45,85],[39,108],[106,91],[45,79],[69,32],[74,84],[52,47]]}